<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-10-05T22:27:32 IST" name="Default suite" finished-at="2023-10-05T22:27:42 IST" duration-ms="9085">
    <groups>
    </groups>
    <test started-at="2023-10-05T22:27:32 IST" name="Default test" finished-at="2023-10-05T22:27:42 IST" duration-ms="9085">
      <class name="timeout.com.TimeOutClass">
        <test-method signature="timeOutTestCase1()[pri:0, instance:timeout.com.TimeOutClass@75d3a5e0]" started-at="2023-10-05T22:27:32 IST" name="timeOutTestCase1" finished-at="2023-10-05T22:27:42 IST" duration-ms="9028" status="FAIL">
          <exception class="org.testng.internal.thread.ThreadTimeoutException">
            <message>
              <![CDATA[Method timeout.com.TimeOutClass.timeOutTestCase1() didn't finish within the time-out 9000]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method timeout.com.TimeOutClass.timeOutTestCase1() didn't finish within the time-out 9000
at java.base@11.0.16.1/java.lang.Thread.sleep(Native Method)
at app//timeout.com.TimeOutClass.timeOutTestCase1(TimeOutClass.java:9)
at java.base@11.0.16.1/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base@11.0.16.1/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base@11.0.16.1/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base@11.0.16.1/java.lang.reflect.Method.invoke(Method.java:566)
at app//org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at app//org.testng.internal.invokers.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:47)
at app//org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:76)
at app//org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:11)
at java.base@11.0.16.1/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base@11.0.16.1/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base@11.0.16.1/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base@11.0.16.1/java.lang.Thread.run(Thread.java:834)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.internal.thread.ThreadTimeoutException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- timeOutTestCase1 -->
      </class> <!-- timeout.com.TimeOutClass -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
